gear/csv_center_splitter/src/python/csv_center_splitter_app/main.py

```diff
@@ -200,29 +204,49 @@ def run(*,
             f"Missing {target_project} projects for the following " +
             f"ADCIDs: {missing_projects}")
 
-    log.info(f"Writing split results for the following ADCIDs: {centers}")
+    if not batch_size:  # just make one big chunk
+        batch_size = len(visitor.split_data)
```

Comment:
At this point, it should actually be an error if there is no batch size. So make the param non-optional